src/main/java/org/zendesk/client/v2/model/Ticket.java

```diff
@@ -218,6 +221,34 @@ public void setIsPublic(Boolean isPublic) {
     this.isPublic = isPublic;
   }
 
+  /**
+   * The safe_update & update_stamp parameters are used by the Zendesk API to protect against update
```

Comment:
I believe the javadoc generation error is due to the unencoded &. Let's try:

```suggestion
   * The safe_update &amp; update_stamp parameters are used by the Zendesk API to protect against update
```